supportedBackends/tes/src/main/scala/cromwell/backend/impl/tes/TesAsyncBackendJobExecutionActor.scala

```diff
@@ -90,6 +168,21 @@ class TesAsyncBackendJobExecutionActor(override val standardParams: StandardAsyn
     )
   }
 
+  override def scriptPreamble: String = {
+    val tesTaskPreamble: String = runtimeAttributes.localizedSasEnvVar.map{enviornmentVariableName =>
+      val workflowName = workflowDescriptor.callable.name
+      val callInputFiles = jobDescriptor.fullyQualifiedInputs.safeMapValues {
+        _.collectAsSeq { case w: WomFile => w }
+      }
+      val taskInputs: List[Input] = TesTask.buildTaskInputs(callInputFiles, workflowName, mapCommandLineWomFile)
+      val preamble = determineWSMSasEndpointFromInputs(taskInputs, getPath).map{ endpoint =>
+        generateLocalizedSasScriptPreamble(enviornmentVariableName, endpoint)
+      }.getOrElse("")
+      preamble
+    }.getOrElse("")
```

Comment:
Do we need two different `getOrElse("")` here? If we use a `flatMap` in line 172 we should be able to leave out the inner one.